src/constants/permissions.ts

```diff
@@ -56,6 +56,11 @@ export const Permissions = {
     UPDATE: 'tasks.update',
     LAUNCH: 'tasks.launch',
   },
+  Runs: {
+    ALL: 'runs.*',
+    DELETE: 'runs.delete',
+    MARK_INELIGIBLE_SCORE: 'runs.mark_ineligible_for_scoring',
```

Comment:
Shouldn't this follow the pattern used in `Users`? 

```suggestion
    Scores: {
      MARK_INELIGIBLE: 'runs.scores.mark_ineligible',
    }
```